### PR TITLE
Fix fedora support, improve unsupported image err

### PR DESCRIPTION
**Pull request by matoval**
##### SUMMARY

Fixes issue https://github.com/ansible-collections/osbuild.composer/issues/14

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: roles/setup_server/tasks/main.yaml [hunk 1]
```diff
@@ -26,8 +26,9 @@
 
 - name: RHEL8 specific setup
```
Comment (maxamillion) [inline roles/setup_server/tasks/main.yaml:27]: This is supposed to be specifically for RHEL8, I don't think Fedora should be included in this conditional block

File: roles/setup_server/tasks/main.yaml [hunk 2]
```diff
@@ -43,8 +44,9 @@
         name:
           - python3-toml
   when:
-    - ansible_distribution in ['RedHat', 'CentOS']
-    - ansible_distribution_major_version|int == 9
+    - ansible_distribution in ['RedHat', 'CentOS', 'Fedora']
+    - ansible_distribution == 'Fedora' and ansible_distribution_major_version|int > 32 or
+      ansible_distribution_major_version|int == 9
```
Comment (maxamillion) [inline roles/setup_server/tasks/main.yaml:49]: This is supposed to be specifically for RHEL9, I don't think Fedora should be included in this conditional block. It's possible Fedora needs it's own separate one or the conditional needs changing and the block name will as well

Comment (matoval) [reply roles/setup_server/tasks/main.yaml:49]: I think renaming the block is the appropriate solution

**Comment by chadmf** [comment]
lgtm!

**Pull request merged**